composer.json

```diff
@@ -17,12 +17,19 @@
         "container-interop/definition-interop": "~0.2.0",
         "container-interop/container-interop": "^1.0",
         "jeremeamia/superclosure": "^2.0",
-        "mouf/picotainer": "^1.0"
+        "mouf/picotainer": "^1.0",
+        "thecodingmachine/service-provider-registry": "^1.0"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/xhuberty/assembly"
```

Comment:
Nope! This is part of the [TODO just above](https://github.com/thecodingmachine/yaco/pull/8#issuecomment-210059415).

By the way, this PR is WIP (work in progress) and therefore should not be merged yet!